.travis.yml

```diff
@@ -9,7 +9,7 @@ bundler_args: --without development
 cache: bundler
 
 before_install:
-  - source script/install_phantomjs.sh
+  - npm install -g phantomjs-prebuilt
```

Comment:
Mmmm. You're right, I copied the trick from discourse, but they must have some extra config to make it work. I assumed it was working because I could no longer see the warning of using an old phantomjs version in the build logs, BUT... the reason seems to be that the build is actually using the `phantomjs` gem (a `jasmine` dependency) to download the binary. The idea was to avoid using that gem because it downloads from bitbucket which is rate limited and fails very often.

Conclusion: thanks for finding this out, I'll think of a solution or revert to the old way of installing it.